visualization-design.typ

```diff
@@ -40,33 +40,40 @@ It is unlikely D3.js will be required for any visualization views other than the
 The visualization will need to recieve an array of balancers:
 
 ```json
-balancers = []
+balancers = [],
 ```
 
-Each balancer must have a state, and from that state, the following is needed:
+Each balancer must have a state pictured below. The group property is not recieved from the balancer, but is one of the properties in json files read by D3.js:
```

Comment:
> The group property

What is this?

> properties in json files read by D3.js

I'm confused -- this is supposed to be the format which is fed to the visualizations right? Why does it matter where it comes from?